api/v1alpha1/devfileregistry_types.go

```diff
@@ -26,25 +27,35 @@ import (
 
 // DevfileRegistrySpec defines the desired state of DevfileRegistry
 type DevfileRegistrySpec struct {
-	// Sets the container image containing devfile stacks to be deployed on the Devfile Registry
-	DevfileIndexImage string `json:"devfileIndexImage,omitempty"`
```

Comment:
Removing this will cause a breaking change.  I know we don't have releases on the operator and we likely don't have clients using this so it _might_ be ok.  We could deprecate this field and remove it once we cut a release.  This will give us time to root out any areas that might be dependent this like e2e tests.